frontend/src/app/patients/Components/PersonForm.tsx

```diff
@@ -244,6 +244,68 @@ const PersonForm = (props: Props) => {
     return schemaOrder;
   };
 
```

Comment:
not part of the scope of this ticket but using react-hook-forms would probably have save us the effort of implementing setting of focus logic